File: app/src/main/java/com/bopr/android/smailer/sync/MetaData.java
package com.bopr.android.smailer.sync;

import com.google.api.client.util.Key;

/**
 * Data transfer object for synchronization metadata.
 */
@SuppressWarnings("WeakerAccess")
public class MetaData {

    @Key("sync_time")
    public long syncTime;
}

File: app/src/main/kotlin/com/bopr/android/smailer/sync/MetaData.kt
package com.bopr.android.smailer.sync

import com.google.api.client.util.Key

/**
 * Data transfer object for synchronization metadata.
 */
/* NOTE: JSON engine requires that all DTO constructor parameters be var and have default value */
data class MetaData(
        @Key("sync_time") var time: Long = 0
)